src/allmydata/uri.py

```diff
@@ -701,7 +712,7 @@ class DirectoryURIVerifier(_DirectoryBaseURI):
 
     BASE_STRING=b'URI:DIR2-Verifier:'
     BASE_STRING_RE=re.compile(b'^'+BASE_STRING)
-    INNER_URI_CLASS=SSKVerifierURI
+    INNER_URI_CLASS=SSKVerifierURI  # type: Type[IVerifierURI]
```

Comment:
I'm not sure I know what this means.  Are there some mypy or mypy-zope docs I should read?  Neither the docstring for `typing.Type` nor the information on https://realpython.com/python-type-checking/ seems to explain what's happening here.

Reply:
I don't have any good references. I've just been picking it up by trial-and-error. What I think this is saying is the `INNER_URI_CLASS` is a `Type` (aka class) that's an instance of the metaclass `IVerifierURI`. That's only really a guess, though. I admit, this approach may not be correct, but it's one solution that unblocks the errors that occur when subclasses of DirectoryURIVerifier indicate a different type for INNER_URI_CLASS. The intention here is to declare some type that's common among all DirectoryURIVerifiers.